src/Umbraco.Configuration/Models/MemberPasswordConfigurationSettings.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.Extensions.Configuration;
+using Umbraco.Core.Configuration;
+
+namespace Umbraco.Configuration.Models
+{
+    internal class MemberPasswordConfigurationSettings : IMemberPasswordConfiguration
```

Comment:
We'll prob need to modify these settings to be inline with identity, but we can deal with that later.